packages/coordinator/src/screens/AuthScreens/ResetPasswordScreen.tsx

```diff
@@ -1,28 +1,36 @@
 import { useState } from "react";
 import styles from "./AuthScreens.module.scss";
 import Input from "../../components/Input";
-import Button from "../../components/Button";
+import Button, { ButtonVariant } from "../../components/Button";
+import emailNotFound from "../../assets/email_not_found.svg";
+import emailOnTheWay from "../../assets/email_on_the_way.svg";
 import logoImage from "./../../assets/logo_drop_ZM.svg";
 import chevronSvg from "./../../assets/icons/chevron-right-small.svg";
 import { useNavigate } from "react-router-dom";
+import { Popup } from "../../components/Popup/Popup";
 
 export interface ResetPasswordScreenProps {
-  onResetPassword: (
-    email: string,
-    emailError: (error: string) => void
-  ) => Promise<void>;
+  onResetPassword: (email: string, emailError: (error: string) => void) => void;
```

Comment:
If you want to await on this, you have to return `Promise<void>`

Reply:
done